src/Selenium2Library/keywords/_element.py

```diff
@@ -296,7 +296,21 @@ def get_horizontal_position(self, locator):
         if element is None:
             raise AssertionError("Could not determine position for '%s'" % (locator))
         return element.location['x']
-
+ 
+    def get_element_size(self, locator):
+        """Returns width and height of element identified by `locator`.
+            
+        The height abd height and width is returned in pixels,as an integer.
```

Comment:
Is is said somewhere that it is pixels and will always be integers? I did try to find it, but could not confirm from anywhere.